### PR TITLE
Language creation (cz)

### PR DESCRIPTION
**Pull request by Heroyt**
about, context, menu_darvin, welcome - fully translated
settings - partialy

This PR/Translation is about #188 

**files translated**:
  - [x] `def/cz/about.cson`
  - [x] `def/cz/context.cson`
  - [x] `def/cz/menu_darwin.cson`
  - [x] `def/cz/welcome.cson`

**validation**
  - [x] I've validated my translation locally by running

`npm run validation -- --locale cz`

### CODE REVIEW

**Comment by liuderchi** [comment]
Thank you @Heroyt for contributing atom-i18n 🚀

Is Czech written as `čeština` in Czech?

File: package.json [hunk 1]
```diff
@@ -113,6 +113,10 @@
           "value": "zh-tw",
           "description": "繁體中文 (zh-tw)"
         },
+        {
+          "value": "cz",
```
Comment (liuderchi) [inline package.json]: Considering alphabetic order,
please move these addition right after `"value": "be",`

Comment (Heroyt) [reply package.json]: Ok, I'll do it

Comment (Heroyt) [reply package.json]: And Czech is čeština in Czech.

Comment (Heroyt) [reply package.json]: @liuderchi I suppose I need to create another pull request for that, am I right?

**Comment by Heroyt** [comment]
@liuderchi It should be ready now.

**Comment by Heroyt** [comment]
@liuderchi Done

**Comment by liuderchi** [comment]
@Heroyt [0.16.0](https://atom.io/packages/atom-i18n) has been Published with Czech support 🎉

**Pull request merged**